xl/trax/track.py

```diff
@@ -395,10 +395,11 @@ def read_tags(self, force=True, notify_changed=True):
 
             # Retrieve file specific metadata
             gloc = Gio.File.new_for_uri(loc)
-            mtime = gloc.query_info(
-                "time::modified", Gio.FileQueryInfoFlags.NONE, None
-            ).get_modification_time()
-            mtime = mtime.tv_sec + (mtime.tv_usec / 100000.0)
+            mtime = (
+                gloc.query_info("time::modified", Gio.FileQueryInfoFlags.NONE, None)
+                .get_modification_date_time()
```

Comment:
According to docs, `get_modification_date_time` was added in GLib 2.62, which was released a year ago. I think this is going to break LTS distributions that use older versions.